implementations/generate.js

```diff
@@ -48,6 +48,7 @@ const deprecatedTests = [
   'Basic Documents Presentations MUST include `verifiableCredential` and `proof` (negative - missing `proof`)',
   'Zero-Knowledge Proofs (optional) A verifiable presentation... MUST include `verifiableCredential`',
   'Zero-Knowledge Proofs (optional) A verifiable presentation... MUST include `verifiableCredential` (negative - missing `verifiableCredential`)',
+  'Refresh Service (optional) each object within `refreshService`... value of `id` MUST be a URL identifying a service endpoint',
```

Comment:
This now requires that `id` be a URL ...

Reply:
Oh, this is just the list of deprecated tests. I've updated the PR text now to mention this.

Adding a test to this list suppresses it from appearing the resulting implementation report web page. Otherwise it would still appear, since it is present in some of the implementation report JSON. files.

Reply:
Alternatively, the implementation files could be search-and-replace edited to rename the test. Maybe this would be okay, considering it's the same test, just with a different name?

Reply:
I don't understand why/how no implementations would report a result for a given test, whether that's "unsupported", "supported and successful", or "supported and failed" — unless the test didn't yet exist, which makes me wonder whether the feature being tested had been spec'ed.

If the feature hadn't been spec'ed at time of test run, the result should probably default to report "unsupported".

If the feature had been spec'ed but the test didn't exist when the implementation was tested, the result should probably default to some shorthand for "test had not been implemented when implementation was tested" — in which case, implementers should be strongly encouraged to run the updated test suite and submit an updated report!

Reply:
I had said 
> If the feature hadn't been spec'ed at time of test run, the result should probably default to report "unsupported".

Re-reading, that default should probably indicate "untested" instead of "unsupported", because being unspec'ed there would have been no test to run, and so no way to know whether supported or not.  This would also apply in cases where "the feature had been spec'ed but the test didn't exist".

Reply:
An implementation can skip sections of the test suite using the "sectionsNotSupported" field in their config.json. Then the tests in that section get the result "no support".

The "untested" result happens occurs when the test was added (or renamed) after the particular implementation's report JSON was generated. Wwith the change in this PR, after at least one implementer re-runs the test and adds their report, all the other implementer columns will read "untested" for this test. e.g. after I re-run the test for Spruce:
![2022-02-09-122538_1198x726_scrot](https://user-images.githubusercontent.com/95347/153258713-8ebd805e-c39e-440f-a913-27f4a1db2e53.png)

If instead I add a dummy test report ("Foo" implementation) that skips the "refresh" section using `sectionsNotSupported`, Foo's column says "no support" since it skipped the section - as in the following:
![2022-02-09-124511_1191x720_scrot](https://user-images.githubusercontent.com/95347/153259388-b3b735d4-8796-47f6-afff-99a29ca9258f.png)

Do you mean to suggest adding new result statuses, to distinguish between current possible different reasons for the "untested" result (and/or "no support")? I'm trying to understand what these would be, and why it may be needed.

Reply:
I'm good with most of what you showed here.

One major difference -- I think every test should have a row of results.  If no implementation has reported a result for that test, _all_ implementations should show "untested" for that test -- in no small part because that may be revealing a feature that is _at risk_ because two successful implementations are required to exit CR! -- so this should lead to reruns by at least two implementations that include support for that feature.